astropy/modeling/tests/test_fitters.py

```diff
@@ -739,9 +740,9 @@ def test_2d_set_axis_2_fitting_with_outlier_removal():
 
     poly_set, filt_z = fitter(poly_set, x, y, z)
 
-    assert_allclose(poly_set.c0_0, [[[0., 1.]]], atol=1e-14)
-    assert_allclose(poly_set.c1_0, [[[1., -0.1]]], atol=1e-14)
-    assert_allclose(poly_set.c0_1, [[[1., 0.2]]], atol=1e-14)
+#     assert_allclose(poly_set.c0_0, [[[0., 1.]]], atol=1e-14)
+#     assert_allclose(poly_set.c1_0, [[[1., -0.1]]], atol=1e-14)
+#     assert_allclose(poly_set.c0_1, [[[1., 0.2]]], atol=1e-14)
```

Comment:
This test does not work any more. Is the `FittingWithOutlierRemoval` fitter broken or the model sets with `model_set_axis=`?

Reply:
Uncommented Hmm, works for me :-)